libs/designer-ui/src/lib/editor/base/index.tsx

```diff
@@ -65,8 +65,6 @@ export interface BaseEditorProps {
   BasePlugins?: BasePlugins;
   initialValue: ValueSegment[];
   children?: React.ReactNode;
-  isTrigger?: boolean;
-  showCallbackTokens?: boolean;
```

Comment:
What does showcallbacktokens set for?

Reply:
previously because http callback was a trigger, we weren't showing the token picker, so this was a flag to let the editor know that if this was a HTTP callback to show the tokenpicker regardless of a trigger. Now this isn't needed cause we'll always show the picker